templates/crd-serviceresolvers.yaml

```diff
@@ -62,7 +62,7 @@ spec:
                     items:
                       type: string
                     type: array
-                  namespaces:
+                  namespace:
```

Comment:
todo: look at what effect his would have

Reply:
Nice catch and thanks for the detailed explanation of the impact!

Reply:
The new helm copier caught it!